capstone/fitness/src/main/java/com/google/sps/servlets/GetDataServlet.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
+
+import com.google.sps.util.DataHandler;
+import com.google.sps.util.*;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.Entity;
+
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/get-data")
+public class GetDataServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    try {
+      Entity user = DataHandler.getUser();
+      String Json = "{";
```

Comment:
There are better ways to build a json string, eg. JSONObject in org.json library